app/Chat/index.tsx

```diff
@@ -80,6 +80,16 @@ export const Chat = () => {
     },
   });
 
+  /*
+   * You can uncomment this stub to replace the actual call
+   * if you get stuck and just need the page to render.
+
+  const generatedTextMutation = {
+    mutate: (opts) => { console.log(opts) }
+  }
+
+   */
```

Comment:
This is a silly little bit of code, but if you go exploring tRPC again and get stuck in a "the component won't even load" state, this stub will let your page load with no query functionality. 🔨